<?xml version="1.0"?>
<launch>
 <node pkg="pylon_camera" name="sol_camera" type="pylon_camera_node" output="screen"> <!--  respawn="true" respawn_delay="30" -->
  <param name="camera_frame" value="/sol_camera" type="string" /> <!-- frame to which the camera is attached -->
  <param name="start_exposure" value="5000" type="int" />
  <param name="use_sequencer" value="true" type="bool" />
  <param name="desired_framerate" value="5.0" type="double" />
  <rosparam param="desired_brightness_seq"> [2000, 7000, 10000] </rosparam>
  <param name="use_brightness" value="true" type="bool" />
  <param name="intrinsic_yaml_string" command="cat $(find pylon_camera)/calib/cam15_f6mm_2014-12-18.yml"/>
 </node>
 <node name="image_view" pkg="image_view" type="image_view" respawn="false" output="screen">
  <remap from="image" to="/sol_camera/image_seq"/>
 </node>
</launch>
